test/unit/org/apache/cassandra/index/sasi/SASICQLTest.java

```diff
@@ -348,4 +351,21 @@ public void testIntCompare() throws Throwable
             }
         }
     }
+
+    @Test
+    public void testInOperator() throws Throwable
+    {
+        createTable("CREATE TABLE %s (pk int primary key, v int);");
+
+        createIndex(String.format("CREATE CUSTOM INDEX ON %%s (v) USING 'org.apache.cassandra.index.sasi.SASIIndex' WITH OPTIONS = {'mode': 'CONTAINS'};"));
```

Comment:
Nit: we don't need `String.format` here:
```suggestion
        createIndex("CREATE CUSTOM INDEX ON %s (v) USING 'org.apache.cassandra.index.sasi.SASIIndex' WITH OPTIONS = {'mode': 'CONTAINS'}");
```
Also, I think that the `contains` mode is relevant for the test so we might slightly simplify to:
```suggestion
        createIndex("CREATE CUSTOM INDEX ON %s (v) USING 'org.apache.cassandra.index.sasi.SASIIndex'");
```